두-정수-사이의-합.js

```diff
@@ -0,0 +1,22 @@
+function solution(a, b) {
+    const num = [];
+    
+    let answer = 0;
+
+    //a와 b가 같을 경우 a 반환, b 반환해도 됨
+    if (a === b){
+        answer = a;
+    }
+    //a와 b가 다를 경우 num 배열 넣기
+    //배열값을 오른차순으로 정렬 하여 작은 수 부터 입력을 받는다.  
+    else {
+        num.push(a, b); //a와 b의 배열 추가
+        num.sort((a, b) => a - b); //내림차순
+ for (let i = num[0]; i <= num[1]; i++) {
+            answer += i;
+        }
+        
+    }
+
+        return answer;
+    }
```

Comment:
앞서 말씀드렸던 것 처럼 과도한 줄바꿈은 오히려 가독성을 떨어지게 만들어요.
변수선언 / 메서드 선언에 나눠 줄바꿈 해주시고
문제 푸실 때는 편하게 푸셔도 괜찮지만 들여쓰기 같은 기본적인 컨벤션은 꼭 지켜서 커밋 보내주세요

가장 좋은 건 문제 풀때부터 지켜주시는게 좋겠죠?

아래는 리팩터링 한 코드이니 보시고 다시 공부해보시면 좋을 것 같아요 
```suggestion
function solution(a, b) {
  const num = [];
  let answer = 0;

  if (a === b) answer = a;
  else {
    num.push(a, b);
    num.sort((a, b) => a - b);
    for (let i = num[0]; i <= num[1]; i++) {
      answer += i;
    }
  }

  return answer;
}
```



+ 추가적으로 최소값, 최댓값 찾아주는 메서드로도 풀이가 가능하니 함께 공부해보면 좋을 것 같습니다
```javascript
function solution(a, b) {
  const minNum = Math.min(a, b);
  const maxNum = Math.max(a, b);
  let answer = 0;

  for (let i = minNum; i < maxNum + 1; i++) {
    answer += i;
  }
  return answer;
}
```